// gamerscoin/lib/commands.js
module.exports = {
  addMultiSigAddress: 'addmultisigaddress',
  addNode: 'addnode', // bitcoind v0.8.0+
  backupWallet: 'backupwallet',
  createMultiSig: 'createmultisig',
  createRawTransaction: 'createrawtransaction', // gamerscoind v0.7.0+
  decodeRawTransaction: 'decoderawtransaction', // gamerscoind v0.7.0+
  dumpPrivKey: 'dumpprivkey',
  encryptWallet: 'encryptwallet',
  getAccount: 'getaccount',
  getAccountAddress: 'getaccountaddress',
  getAddedNodeInfo: 'getaddednodeinfo', // gamerscoind v0.8.0+
  getAddressesByAccount: 'getaddressesbyaccount',
  getBalance: 'getbalance',
  getBestBlockHash: 'getbestblockhash', // gamerscoind v0.9.0+
  getBlock: 'getblock',
  getBlockCount: 'getblockcount',
  getBlockHash: 'getblockhash',
  getBlockTemplate: 'getblocktemplate', // gamerscoind v0.7.0+
  getConnectionCount: 'getconnectioncount',
  getDifficulty: 'getdifficulty',
  getGenerate: 'getgenerate',
  getHashesPerSecond: 'gethashespersec',
  getHashesPerSec: 'gethashespersec',
  getInfo: 'getinfo',
  getMiningInfo: 'getmininginfo',
  getNetTotals: 'getnettotals',
  getNetworkInfo: 'getnetworkinfo', // gamerscoind v0.9.2+
  getNetworkHashPs: 'getnetworkhashps', // gamerscoind v0.9.0+
  getNewAddress: 'getnewaddress',
  getPeerInfo: 'getpeerinfo', // gamerscoind v0.7.0+
  getRawMemPool: 'getrawmempool', // gamerscoind v0.7.0+
  getRawTransaction: 'getrawtransaction', // gamerscoind v0.7.0+
  getReceivedByAccount: 'getreceivedbyaccount',
  getReceivedByAddress: 'getreceivedbyaddress',
  getTransaction: 'gettransaction',
  getTxOut: 'gettxout', // bitcoind v0.7.0+
  getTxOutSetInfo: 'gettxoutsetinfo', // gamerscoind v0.7.0+
  getWalletInfo: 'getwalletinfo', // gamerscoind v0.9.2+
  getWork: 'getwork',
  help: 'help',
  importPrivKey: 'importprivkey',
  keypoolRefill: 'keypoolrefill',
  keyPoolRefill: 'keypoolrefill',
  listAccounts: 'listaccounts',
  listAddressGroupings: 'listaddressgroupings', // gamerscoind v0.7.0+
  listLockUnspent: 'listlockunspent', // gamerscoind v0.8.0+
  listReceivedByAccount: 'listreceivedbyaccount',
  listReceivedByAddress: 'listreceivedbyaddress',
  listSinceBlock: 'listsinceblock',
  listTransactions: 'listtransactions',
  listUnspent: 'listunspent', // gamerscoind v0.7.0+
  lockUnspent: 'lockunspent', // gamerscoind v0.8.0+
  move: 'move',
  sendFrom: 'sendfrom',
  sendMany: 'sendmany',
  sendRawTransaction: 'sendrawtransaction', // gamerscoind v0.7.0+
  sendToAddress: 'sendtoaddress',
  setAccount: 'setaccount',
  setGenerate: 'setgenerate',
  setTxFee: 'settxfee',
  signMessage: 'signmessage',
  signRawTransaction: 'signrawtransaction', // gamerscoind v0.7.0+
  stop: 'stop',
  submitBlock: 'submitblock', // gamerscoind v0.7.0+
  validateAddress: 'validateaddress',
  verifyChain: 'verifychain', // gamerscoind v0.9.0+
  verifyMessage: 'verifymessage',
  walletLock: 'walletlock',
  walletPassphrase: 'walletpassphrase',
  walletPassphraseChange: 'walletpassphrasechange'
};
